ui/dashboards/src/components/AddPanel/AddPanel.tsx

```diff
@@ -99,14 +118,26 @@ const AddPanel = ({ isOpen, onClose }: AddPanelProps) => {
           </Grid>
           <Grid item xs={8}>
             <Stack spacing={2} sx={{ flexGrow: '1' }}>
-              <FormControl>
-                <TextField required label="Panel Name" variant="outlined" onChange={onPanelNameChange} />
-              </FormControl>
-              <FormControl>
-                <TextField label="Description" variant="outlined" onChange={onPanelDescriptionChange} />
-              </FormControl>
+              <TextField required label="Panel Name" variant="outlined" onChange={handlePanelNameChange} />
+              <TextField label="Description" variant="outlined" onChange={handlePanelDescriptionChange} />
             </Stack>
           </Grid>
+          <Grid item xs={4}>
+            <FormControl>
+              <InputLabel id="panel-type-label">Panel Type</InputLabel>
+              <Select required labelId="panel-type-label" label="Panel Type" value={kind} onChange={handleKindChange}>
+                {/* TODO: Replace this with options that come from asking the plugin system what panel plugins are available */}
```

Comment:
YAY I was wondering about this

Reply:
I was like, eventually its gotta come from asking somewhere for the data about what kinds of panels are available to some app...